frontend/src/components/TaskMapEdge.module.scss

```
@@ -44,25 +44,13 @@
 .menuItem {
   background-color: $COLOR_WHITE !important; // !important to override MUI default styles
   color: $COLOR_BLACK10; // Adds color for default effect from MUI: turn background color of the menu item to lower shade of the "color" on click
-  width: 48px;
-  &:hover {
-    transition: width 0.4s;
-    width: 185px;
-  }
+  width: 185px;
```

Other (if we had some) translations could take up more space, so changing to max-content would be a better practice
On the other hand, I did not see removing width changing anything, so is it required at all?
```suggestion
  width: max-content;
```

---

Ah, that's a good point. Width seems to be defaulted at max-content so removing it altogether should be good. I'll do the change and add translation for the "remove relation" as well, seems like it was missing 👍